src/components/Reports/SubscriberHistory/SubscriberHistory.test.js

```diff
@@ -236,4 +241,57 @@ describe('SubscriberHistory component', () => {
       expect(getByText('subscriber_history.delivery_status.notOpened')).toBeInTheDocument();
     });
   });
+
+  it('should load campaign preview img', async () => {
```

Comment:
Perhaps add another test for when tooltip is not found?

Reply:
ready